src/cascadia/TerminalApp/AppKeyBindings.cpp

```diff
@@ -377,6 +377,18 @@ namespace winrt::TerminalApp::implementation
             _MoveFocusHandlers(*this, *eventArgs);
             return eventArgs->Handled();
         }
+        case ShortcutAction::IncreaseFontSize:
+        {
+            auto eventArgs = winrt::make_self<ActionEventArgs>();
+            _IncreaseFontSizeHandlers(*this, *eventArgs);
+            return eventArgs->Handled();
+        }
+        case ShortcutAction::DecreaseFontSize:
+        {
+            auto eventArgs = winrt::make_self<ActionEventArgs>();
+            _DecreaseFontSizeHandlers(*this, *eventArgs);

```

Comment:
Worth it to combine them into some kind of AdjustFontSize event and pass the amount to change in event args

Reply:
Oh, absolutely. If we put the delta into an args, somebody could (in the future) bind +5, +10, -15, etc.

Reply:
(@zadjii-msft)